spec/strategy_spec.rb

```diff
@@ -163,11 +163,11 @@ def identity
     end
 
     it 'should update user-records with retrieved information' do
-      User.should have(1).record
+      expect(User.count).to eq 1
       User.first.email.should == 'dimitrij@example.com'
     end
   end
-  
+
   describe "POST /users/sign_in (from OpenID provider, success, rememberable)" do
 
     before do
```

Comment:
context: Gemfile.devise33 (working)

Here below at line 175, before the controller action is executed, `User.first.remember_token.present? == true`

context: Gemfile.devise35 (NOT working)

At line 175 `User.first.remember_token.present? == false` (already).
If it is assigned, the build passes.

Something in the creation of the user changed.